kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/PythonClient.java

```diff
@@ -0,0 +1,133 @@
+    };
+    private String deployNamespace;
+
+    @Override
+    public KafkaClient inNamespace(String namespace) {
+        this.deployNamespace = namespace;
+        return this;
+    }
+
+    /**
+     * Instantiates a new python client.
+     */
+    public PythonClient() {
+        this.deployNamespace = kubeClient().getNamespace();
+    }
+
+    @Override
+    public void produceMessages(String topicName, String bootstrap, String message, @Nullable String messageKey, int numOfMessages) {
+        final Optional<String> recordKey = Optional.ofNullable(messageKey);
+
+        StringBuilder msg = new StringBuilder();
+        for (int i = 0; i < numOfMessages; i++) {
+            msg.append(message)
+                    .append(" - ")
+                    .append(i)
+                    .append("\n");
+        }
+
+        LOGGER.atInfo().setMessage("Producing messages in '{}' topic using python").addArgument(topicName).log();
+        String name = Constants.KAFKA_PRODUCER_CLIENT_LABEL + "-python-" + TestUtils.getRandomPodNameSuffix();
+        String jsonOverrides = KubeUtils.isOcp() ? TestUtils.getJsonFileContent("nonJVMClient_openshift.json").replace("%NAME%", name) : "";
+
+        List<String> executableCommand = new ArrayList<>(List.of(cmdKubeClient(deployNamespace).toString(), "run", "-i",
+                "-n", deployNamespace, name,
+                "--image=" + Constants.PYTHON_CLIENT_IMAGE,
+                "--override-type=strategic",
+                "--overrides=" + jsonOverrides,
+                "--", "python3", "/usr/src/confluent-kafka-python/Producer.py", "-b", bootstrap, "-t", topicName));
+        recordKey.ifPresent(key -> {
+            executableCommand.add("-k");
+            executableCommand.add(key);
+        });
+
+        KafkaUtils.produceMessagesWithCmd(deployNamespace, executableCommand, String.valueOf(msg), name, KafkaClientType.PYTHON.name().toLowerCase());
+    }
+
+    @Override
+    public List<ConsumerRecord> consumeMessages(String topicName, String bootstrap, int numOfMessages, Duration timeout) {
+        LOGGER.atInfo().log("Consuming messages using python");
+        String name = Constants.KAFKA_CONSUMER_CLIENT_LABEL + "-python-" + TestUtils.getRandomPodNameSuffix();
+        // Running consumer with parameters to get the latest N number of messages received to avoid consuming twice the same messages
+        List<String> args = List.of("python3", "/usr/src/confluent-kafka-python/Consumer.py", "-n", String.valueOf(numOfMessages), "-b", bootstrap, "-t", topicName);
+        Job pythonClientJob = TestClientsJobTemplates.defaultPythonJob(name, args).build();
+        String podName = KafkaUtils.createJob(deployNamespace, name, pythonClientJob);
+        String log = waitForConsumer(deployNamespace, podName, timeout);
+        KafkaUtils.deleteJob(pythonClientJob);
+        LOGGER.atInfo().setMessage("Log: {}").addArgument(log).log();
```

Comment:
I think we should make this debug level (prior to merging) and change the prefix e.g. 

```suggestion
        LOGGER.atInfo().setMessage("Pod STD_OUT: {}").addArgument(log).log();
```
As log could also mean the Kafka message log.

Reply:
I agree with replacing `Log` but `Pod stdout` but keep it at info level as it is really easy to see if everything works fine when watching the received messages in the test log.